src/index.html

```diff
@@ -17,6 +17,88 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="logo_link"
+        href="index.html"
+      >
+        <img
+          class="logo_img"
+          src="./images/logo.png"
+          alt="logo"
```

Comment:
The `alt` attribute should describe the image content. Instead of a generic 'logo', provide a more descriptive text that reflects what the logo represents or the company/brand it belongs to.